vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -1199,86 +1199,89 @@ void LedgerImpl::SetCatalogIssuers(const std::string& info) {
   bat_confirmations_->SetCatalogIssuers(std::move(issuers_info));
 }
 
-void LedgerImpl::ConfirmAd(const std::string& info) {
-  ads::NotificationInfo notification_info_ads;
-  if (notification_info_ads.FromJson(info) != ads::Result::SUCCESS)
+void LedgerImpl::ConfirmAdNotification(const std::string& info) {
+  ads::AdNotificationInfo notification_info_ads;
+  if (notification_info_ads.FromJson(info) != ads::Result::SUCCESS) {
     return;
+  }
 
-  auto notification_info = std::make_unique<confirmations::NotificationInfo>();
-  notification_info->id = notification_info_ads.id;
-  notification_info->parent_id = notification_info_ads.parent_id;
+  auto notification_info =
+      std::make_unique<confirmations::AdNotificationInfo>();
+  notification_info->uuid = notification_info_ads.uuid;
+  notification_info->parent_uuid = notification_info_ads.parent_uuid;
+  notification_info->creative_instance_id =
+      notification_info_ads.creative_instance_id;
   notification_info->creative_set_id = notification_info_ads.creative_set_id;
   notification_info->category = notification_info_ads.category;
-  notification_info->advertiser = notification_info_ads.advertiser;
-  notification_info->text = notification_info_ads.text;
-  notification_info->url = notification_info_ads.url;
-  notification_info->uuid = notification_info_ads.uuid;
-
-  switch (notification_info_ads.type.value()) {
-    case ads::ConfirmationType::UNKNOWN: {
-      notification_info->type = confirmations::ConfirmationType::UNKNOWN;
+  notification_info->title = notification_info_ads.title;
+  notification_info->body = notification_info_ads.body;
+  notification_info->target_url = notification_info_ads.target_url;
```

Comment:
Suggest: AdNotificationInfo constructor to consume notification_info_ads & creative_instance

Reply:
This PR is to mainly rename functions/variables ready for Publisher Ads, so will add to a future refactor ticket